nilearn/plotting/img_plotting.py

```diff
@@ -556,6 +559,159 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
+        figure : integer or matplotlib figure, optional
+            Matplotlib figure used or its number. If None is given, a
+            new figure is created.
+        axes : matplotlib axes or 4 tuple of float: (xmin, ymin, width, height), optional
+            The axes, or the coordinates, in matplotlib figure space,
+            of the axes used to display the plot. If None, the complete
+            figure is used.
+        title : string, optional
+            The title displayed on the figure.
+        annotate: boolean, optional
+            If annotate is True, positions and left/right annotation
+            are added to the plot.
+        draw_cross: boolean, optional
+            If draw_cross is True, a cross is drawn on the plot to
+            indicate the cut plosition.
+        black_bg: boolean, optional
+            If True, the background of the image is set to be black. If
+            you wish to save figures with a black background, you
+            will need to pass "facecolor='k', edgecolor='k'" to pylab's
+            savefig.
+        cmap: matplotlib colormap, optional
+            The colormap for the atlas maps
+        vmin: float
+            Lower bound for plotting, passed to matplotlib.pyplot.imshow
+        vmax: float
+            Upper bound for plotting, passed to matplotlib.pyplot.imshow
+        alpha: float between 0 and 1
+            Alpha value sets the transparency of the color fillings inside the
+            contours.
+    """
+    display = plot_anat(anat_img, cut_coords=cut_coords,
+                        display_mode=display_mode,
+                        figure=figure, axes=axes, title=title,
+                        annotate=annotate, draw_cross=draw_cross,
+                        black_bg=black_bg, **kwargs)
+
+    maps_img = _utils.check_niimg_4d(maps_img)
+    n_maps = maps_img.shape[3]
+
+    allowed_view_types = list(('auto', 'contours', 'filled_contours', 'continuous'))
+    if not isinstance(view_type, _basestring) or \
+            view_type not in allowed_view_types:
+        message = ('view_type option should be given '
+                   'either of these {0} ').format(allowed_view_types)
+        raise ValueError(message)
+
+    cmap = plt.cm.get_cmap(cmap)
+    # Build a custom colormap for displaying contours
+    color_list = cmap(np.linspace(0, 1, n_maps))
+
+    if threshold is None:
+        # it will use default percentage,
+        # for a nicer look avoiding maximum overlaps for visualization
+        threshold = "99.7%"
+
+    if isinstance(threshold, list):
+        if len(threshold) != n_maps:
+            raise TypeError('The list of values to threshold '
+                            'should be equal to number of maps')
+    else:
+        threshold = [threshold] * n_maps
+
+    if view_type == 'auto':
+        if n_maps > 4:
+            view_type = 'contours'
+        else:
+            view_type = 'continuous'
+    elif view_type == 'contours':
```

Comment:
Should I change ```elif ``` to ```if``` ?
> otherwise filled will not be properly initialized for auto

We were discussed not to include filled in ```auto```. Sorry I could not get it properly.